src/RegistrationController.js

```diff
@@ -198,6 +198,22 @@ export default BaseLoginController.extend({
         noCancelButton: true,
         title: loc('registration.form.title', 'login'),
         save: loc('registration.form.submit', 'login'),
+        initialize () {
```

Comment:
If we can't find a better to do this early morning.. then i am fine with this approach.

we can add
`
modelEvents : {
'invalid error' : 'showErrors'
}`

and let showError handle the logic..

Reply:
until if @haishengwu-okta or @magizh-okta have any better idea..

Reply:
@pradeepdewda-okta `showErrors` is not available to the Form, so I would need to add another Form method to call the controller.showErrors. And the invalid error object for the form field is different than the ajax error object showErrors expects.

Reply:
@sherizada-okta add a new handler.. not the base showError method. sorry for the confusion.